2021-08-02 13:17:37,482 INFO    MainThread:79672 [wandb_setup.py:_flush():69] setting env: {}
2021-08-02 13:17:37,482 INFO    MainThread:79672 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-02 13:17:37,482 INFO    MainThread:79672 [wandb_init.py:_log_setup():337] Logging user logs to /home/nils/Documents/Studium/Bachelorarbeit/moveo_RL/wandb/run-20210802_131737-38bifcsq/logs/debug.log
2021-08-02 13:17:37,482 INFO    MainThread:79672 [wandb_init.py:_log_setup():338] Logging internal logs to /home/nils/Documents/Studium/Bachelorarbeit/moveo_RL/wandb/run-20210802_131737-38bifcsq/logs/debug-internal.log
2021-08-02 13:17:37,482 INFO    MainThread:79672 [wandb_init.py:init():370] calling init triggers
2021-08-02 13:17:37,482 INFO    MainThread:79672 [wandb_init.py:init():375] wandb.init called with sweep_config: {}
config: {}
2021-08-02 13:17:37,483 INFO    MainThread:79672 [wandb_init.py:init():419] starting backend
2021-08-02 13:17:37,483 INFO    MainThread:79672 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-02 13:17:37,493 INFO    MainThread:79672 [backend.py:ensure_launched():135] starting backend process...
2021-08-02 13:17:37,499 INFO    MainThread:79672 [backend.py:ensure_launched():139] started backend process with pid: 79691
2021-08-02 13:17:37,500 INFO    MainThread:79672 [wandb_init.py:init():424] backend started and connected
2021-08-02 13:17:37,501 INFO    MainThread:79672 [wandb_init.py:init():472] updated telemetry
2021-08-02 13:17:37,502 INFO    MainThread:79672 [wandb_init.py:init():491] communicating current version
2021-08-02 13:17:38,206 INFO    MainThread:79672 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.11.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-02 13:17:38,206 INFO    MainThread:79672 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-08-02 13:17:38,468 INFO    MainThread:79672 [wandb_init.py:init():529] starting run threads in backend
2021-08-02 13:17:42,151 INFO    MainThread:79672 [wandb_run.py:_console_start():1624] atexit reg
2021-08-02 13:17:42,151 INFO    MainThread:79672 [wandb_run.py:_redirect():1498] redirect: SettingsConsole.REDIRECT
2021-08-02 13:17:42,152 INFO    MainThread:79672 [wandb_run.py:_redirect():1503] Redirecting console.
2021-08-02 13:17:42,153 INFO    MainThread:79672 [wandb_run.py:_redirect():1559] Redirects installed.
2021-08-02 13:17:42,153 INFO    MainThread:79672 [wandb_init.py:init():554] run started, returning control to user process
2021-08-02 13:17:42,155 INFO    MainThread:79672 [wandb_run.py:_atexit_cleanup():1594] got exitcode: 1
2021-08-02 13:17:42,155 INFO    MainThread:79672 [wandb_run.py:_restore():1566] restore
2021-08-02 13:17:44,875 INFO    MainThread:79672 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 859
  total_bytes: 859
}

2021-08-02 13:17:45,488 INFO    MainThread:79672 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 859
  total_bytes: 861
}

2021-08-02 13:17:45,815 INFO    MainThread:79672 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9905
  total_bytes: 9905
}

2021-08-02 13:17:45,917 INFO    MainThread:79672 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9905
  total_bytes: 9905
}

2021-08-02 13:17:46,018 INFO    MainThread:79672 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9905
  total_bytes: 9905
}

2021-08-02 13:17:46,119 INFO    MainThread:79672 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9905
  total_bytes: 9905
}

2021-08-02 13:17:46,220 INFO    MainThread:79672 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9905
  total_bytes: 9905
}

2021-08-02 13:17:46,322 INFO    MainThread:79672 [wandb_run.py:_wait_for_finish():1716] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9905
  total_bytes: 9905
}

2021-08-02 13:17:47,753 INFO    MainThread:79672 [wandb_run.py:_show_files():1938] logging synced files
